config/urls.py

```diff
@@ -38,7 +38,7 @@
     path('admin/cms-extras/', decorator_include(skip_ga360, cms_extras.urls, namespace='cms_extras')),
     path('admin/', decorator_include(skip_ga360, wagtailadmin_urls)),
     path('documents/', include(wagtaildocs_urls)),  # NB: doesn't skip GA as we may analytics on this
-    path('sso/', include(sso.urls)),
+    path('great-cms-sso/', include(sso.urls)),
```

Comment:
Minor nit: the only thing i don't like about this is that it reveals a nonsensical name to the outside world.

Suggesting something less codebase-specific, but ignore if you want:

```suggestion
    path('sso-internal/', include(sso.urls)),
```